zubhub_frontend/zubhub/src/views/PageWrapper.jsx

```diff
@@ -302,14 +302,14 @@ function PageWrapper(props) {
               {t('pageWrapper.footer.privacy')}
             </Typography>
 
-            <Link to={`/privacy_policy`} className={common_classes.textDecorationNone}>
-              <Typography variant="subtitle2" color="textPrimary" className={classes.footerLinkStyle}>
+            <Link to={`/privacy_policy`} >
+              <Typography variant="subtitle2" color="textPrimary" className={classes.footerLinkStyle} style={{ textDecoration: 'underline' }}>
```

Comment:
We basically use `JSS` for styling and I think we should try to keep that convention if possible. Try to look for a file that might be called `pageWrapperStyles.js` and apply your changes there.